tests/test_geoprocessing.py

```diff
@@ -4877,3 +4877,55 @@ def test_iterblocks_bad_raster(self):
         expected_message = 'could not be opened'
         actual_message = str(cm.exception)
         self.assertTrue(expected_message in actual_message, actual_message)
+
+    def test_warp_raster_signedbyte(self):
+        """PGP.geoprocessing: warp raster test."""
+        import pygeoprocessing
+        import pygeoprocessing.testing
+
+        workspace_dir = 'test_workspace'
+        try:
+            os.makedirs(workspace_dir)
+        except OSError:
+            pass

```

Comment:
How about using `self.workspace_dir` in this test instead? We already have setup and teardown methods for it in the class.

Reply:
changed in 13a9e8b8e38369d40d866f1a46aaa7f3d551009c